stories/Divider.stories.mdx

```diff
@@ -0,0 +1,39 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
+import { linkTo } from '@storybook/addon-links';
+import { Divider } from '../src'; // components you used in the story
+
+<Meta title="Layout/Divider" component={Divider} />
+
+# Divider
+
+Divider component serves as an hr for ease of templating. A very simple component 
+but can add a lot in the experience of the user. It can be used to seperate between sections.
+
+## Table of Contents
+
+In this documentation we talk about the Divider component in depth about its usage and its props.
+
+- [Functionality](#functionality)
+- [Other Props & Functionalities](#other-props--functionalities)
+
+## Functionality
+
+Functionally this component will render a divider with the given `color` and `height`.
```

Comment:
talk about the `color` and `height` in a sub-section of `# Functionality` called `## Styling` and show a divider with adjusted color and height in a Canvas to better explain how they work. Should look something like: 

```mdx
## Functionality

The Divider is used like an `hr` element that visually separates elements

<Canvas>
	<Story name="Divider Component" args={{ color: 'lightgray', height: '1' }}>
		{(args) => (
			<Divider {...args} />
		)}
	</Story>
</Canvas>

### Styling

Description of styling options

<Canvas>
	<Story name="Divider Styling" args={{ ...differentValues }}>
		{(args) => (
			{ example here }
		)}
	</Story>
</Canvas>

```